### ISSUE TITLE
shortcodes: allow markdown for title and content

### ISSUE THREAD

**Issue by cagix** [McShelby/hugo-theme-relearn#502]
Using a `button` shortcode with markdown content results in a correctly translated but somehow escaped format.

Example: When I use `{{% button %}}Get **Hugo** {{% /button %}}`, the result currently is
```
<span class="btn cstyle transparent">
  <button type="button">
    Get &lt;strong&gt;Hugo&lt;/strong&gt;
  </button>
</span>
```

Expected would be
```
<span class="btn cstyle transparent">
  <button type="button">
    Get <strong>Hugo</strong>
  </button>
</span>
```

This worked in an earlier version of Hugo Relearn theme.

Version: Hugo Relearn 5.12.2, Hugo v0.111.2

**Comment by McShelby** [comment]
Thankg cagix for reporting this. It's fixed in 5.12.3

**Comment by cagix** [comment]
txh!
